deployment/terraform/iam.tf

```diff
@@ -153,6 +153,26 @@ resource "aws_iam_role_policy" "ses_send_email_from_batch" {
   policy = data.aws_iam_policy_document.ses_send_email.json
 }
 
+data "aws_iam_policy_document" "batch_s3_read_policy" {
```

Comment:
Why this changes needed, to get file?

Reply:
Specifying the appropriate IAM policy for AWS Batch jobs is essential to ensure they have the necessary permissions to securely access S3 resources.
In AWS Batch jobs, we run different containers with different IAM permissions compared to the main ECS instances.

Reply:
to get file?

> Yes, to get file from S3